scania-digital-design-system/tegel
# docs(popover-menu): not closing on click outside if show is set.

**Describe pull-request**  
Added docs explaining that the popover does not close on click outside if show is set.

**Solving issue**  
Fixes: [DTS-2157](https://tegel.atlassian.net/browse/DTS-2157)

**How to test**  
1. Go to storybook
2. Check in Popover Menu
3. Check the docs for the show prop.


[DTS-2157]: https://tegel.atlassian.net/browse/DTS-2157?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
